### PR TITLE
Move theming support from Beta to Main

### LINKED ISSUE #57: Update color tokens for theming support
### General Support Request

In order to make Auro components themeable, we need to make sure there are no hard-coded and correct semanticly named tokens within the component. There also needs to be investigation around `border` rules which have a combination of size and color tokens, making them more complex, as well as `background-color` and `color` rules.

### Possible Solution

Work with designers to figure out what tokens need updated values or renamed.

### Additional context

This work stems from theming work happening to all Auro components:
- https://github.com/AlaskaAirlines/AuroDesignTokens/issues/147

Rules and tokens to be edited can be found in the comments.

### Acceptance Criteria:

- [ ] All tier 3 token names have been implemented.
- [ ] Colors identified below have been updated to their tier 3 token name.

### PR DESCRIPTION
**Pull request by jordanjones243**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Resolves:** #57 

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

## Type of change:

Please delete options that are not relevant.

- [x] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

## Summary by Sourcery

Move theming support for the auro-banner component from beta to main, enabling restyling through design tokens. Refactor color tokens to align with tier 3 standards and update documentation to reflect these changes.

New Features:
- Introduce theme support for the auro-banner component, allowing it to be restyled using new design tokens.

Enhancements:
- Refactor color tokens to use tier 3 tokens for improved consistency and maintainability.

Documentation:
- Update documentation to include information on the new theme support and how to use design tokens for restyling the auro-banner component.

### CODE REVIEW

File: src/tokens.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+@import "./../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables";
+
+:host {
+  --ds-auro-banner-text-color: var(--ds-color-text-primary-default, #{$ds-color-text-primary-default});
```
[the remark this answers is not in the document]
Comment (jordanjones243) [reply src/tokens.scss:4]: Not adding a comment at this time.

File: docs/partials/api.md [hunk 2]
```diff
@@ -158,3 +158,10 @@ The following example illustrates a `<auro-banner>` custom element with the `rou
 <!-- AURO-GENERATED-CONTENT:END -->
```
[the remark this answers is not in the document]
Comment (jordanjones243) [reply docs/partials/api.md:158]: It is indeed rendered on the API page.

File: CHANGELOG.md
Comment (jason-capsule42) [inline CHANGELOG.md:1]: Remove the beta entries from the changelog

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]

**Review by jason-capsule42 — APPROVED** [review]

**Pull request merged**